### DISCUSSION TITLE
Warning en los imports al hacer verify

### DISCUSSION THREAD

**Discussion by mbeorlegui** [dds-jv/foro#359] [Consultas de herramientas] [answered]
Buenas, estoy teniendo un problema con el validador de formato de maven, que es que me exige que los `import` esten todos juntos, cuando IntelliJ me los separa al formatear el código. Intenté configurarle el checkstyle de Google por defecto pero no pude.

Un ejemplo de una linea donde me tira el error es el siguiente:
```java
import domain.miembro.Miembro;
import domain.ubicacion.Ubicacion;

import java.util.ArrayList;
import java.util.List;
import java.util.stream.Collectors;
```
Ahi estan separados los imports de domain y de Java, pero maven me exige que esten juntos (además, estos warnings hacen que no me corran las demas validaciones).

El warning que arroja Maven es el siguiente:
```java
[WARNING] src/main/java/domain/miembro/Miembro.java:[5] (imports) CustomImportOrder: Separación extra en el grupo de importación antes 'java.util.ArrayList'
```

Luego, vi que lo puedo ignorar el warning agregandolo al tag `violationIgnore` que usa maven, y eso me parecio una solucion bastante potable, pero que necesitaría de su aprobación.
El tag quedaría así:
```xml
<violationIgnore>
    MissingJavadocMethod,
    MissingJavadocPackage,
    MissingJavadocType,
    NonEmptyAtclauseDescription,
    JavadocParagraph,
    SummaryJavadoc,
    CustomImportOrder <!-- Linea que agregue -->
</violationIgnore>
```

Aguardo su confirmación para seguir adelante con el cambio este, o si no alguna otra cosa que pueda hacer para evitar el warning.
Muchas gracias y feliz dia del trabajador!

**Comment by mbeorlegui** [comment, marked as answer]
Buenas @gastonprieto sigo pendiente de la respuesta esta, porque ya estamos para entregar, creo que se te pasó porque respondiste todo el resto.
Muchas gracias!

**Reply by mbeorlegui** [reply]
**UPDATE**: Al agregar eso que dije en el `pom.xml`, corrieron bien los verify de los tests de github, así que pareciers que no hay problema con agregarlo.

**Reply by gastonprieto** [reply]
esta bien lo que agregaste @mbeorlegui
